pkg/scheduler/plugins/deviceshare/devicehandler_gpu.go

```diff
@@ -70,6 +71,9 @@ func calcDesiredRequestsAndCountForGPU(podRequests corev1.ResourceList) (corev1.
 	if coreExists {
 		requests[apiext.ResourceGPUCore] = *resource.NewQuantity(gpuCore.Value()/desiredCount, resource.DecimalSI)
 	}
+	if huaweiNPUCoreExists {
```

Comment:
Why does DeviceShare need to support NPUCore allocation? Shouldn't NPU Core just trigger Huawei's DP?